Pcap++/header/PcapFilter.h

```diff
@@ -69,6 +69,18 @@ namespace pcpp
 		LESS_OR_EQUAL
 	} FilterOperator;
 
+	namespace detail
```

Comment:
maybe rename this to `internal` as we have in [other places](https://github.com/seladb/PcapPlusPlus/blob/4d48615c9bfb2ef0c61d50e1e1dd3e3d63a82be0/Common%2B%2B/header/IpUtils.h#L59)?

Reply:
We also have `detail` here: https://github.com/seladb/PcapPlusPlus/blob/4d48615c9bfb2ef0c61d50e1e1dd3e3d63a82be0/Pcap%2B%2B/header/PcapFilter.h#L542

it's better to unify them.

Reply:
yes I agree! I personally like `internal` better than `detail`

Reply:
Implemented 539201b